src/content/docs/new-relic-solutions/new-relic-one/install-configure/update-new-relic-agent.mdx

```diff
@@ -110,7 +110,7 @@ See the docs for your agent:
 
 * **Android**: [Install](/docs/mobile-apps/android-installation-and-configuration) \| [Release notes](/docs/release-notes/mobile-release-notes/android-release-notes)
 * **iOS**: [Install](/docs/mobile-apps/ios-installation-and-configuration) \| [Release notes](/docs/release-notes/mobile-release-notes/ios-release-notes)
-* **tvOS**: [Update](/docs/mobile-monitoring/new-relic-mobile-ios/tvos/upgrading-new-relic-mobiles-tvos-sdk) \| [Release notes](/docs/release-notes/mobile-release-notes/tvos-release-notes)
+* **tvOS**: [Update](/docs/mobile-monitoring/new-relic-mobile-ios/tvos/new-relic-tvos-compatibility-requirements) \| [Release notes](/docs/release-notes/mobile-release-notes/tvos-release-notes)
```

Comment:
* **iOS/tvOS**: [Install](/docs/mobile-apps/ios-installation-and-configuration) \| [Release notes](/docs/release-notes/mobile-release-notes/ios-release-notes)
@mbruin-NR What do you think about this suggestion?

Reply:
This change has been made in commit 01bcdbfced193e38a986a5d6aef0e5ad68350ada [01bcdbfced]